src/linklayer/ext/cSocketRTScheduler.cc

```diff
@@ -220,7 +227,9 @@ bool cSocketRTScheduler::receiveWithTimeout()
 
     found = false;
     timeout.tv_sec = 0;
-    timeout.tv_usec = PCAP_TIMEOUT * 1000;
+    timeout.tv_usec = 100;
```

Comment:
Maybe it is better to use some #define the the timeout. Then is it easier to change parameters, if needed.